_content/booklets/kalama-sutta-free-inquiry_soma-thera.md

```diff
@@ -0,0 +1,21 @@
+---
+title: "Kālāma Sutta: The Buddha's Charter of Free Inquiry"
+authors:
+  - "Soma Thera"
+external_url: "https://www.bps.lk/olib/wh/wh008_Soma_Buddhas-Charter-of-Free-Inquiry.pdf"
```

Comment:
Let's link to the `.html` instead (if people want the pdf or epub they can click the drive links)